torchao/quantization/quant_primitives.py

```diff
@@ -888,14 +888,19 @@ def _choose_qparams_affine(
                 "preserve_zero == False is not supported for symmetric quantization"
             )
         if (
-            zero_point_domain is not None
+            zero_point_domain != ZeroPointDomain.NONE.name
+            and zero_point_domain != None
```

Comment:
I feel we can probably remove support for None since it's the same as ZeroPointDomain.NONE.name

Reply:
Thanks again for reviewing!

Some other places in the codebase are also using both `ZeroPointDomain.NONE.name` and None separately:

https://github.com/pytorch/ao/blob/31234dbc3210a0833cd922856107dc004d088ce1/torchao/quantization/quant_primitives.py#L543

I unified these two cases as one in the latest commit, but I'm not sure if changes in `__tensor_unflatten__` & `__tensor_flatten__` methods of some classes may be required at some other places in the codebase to ensure that they can deal with a `None` zero-point when TorchDynamo would be used . I'll run CUDA-only UTs at my end tomorrow morning to verify.